src/Service/Helper.php

```diff
@@ -68,61 +60,9 @@ class Helper
     private $loggerFactory;
 
     public function __construct(LoggerFactory $loggerFactory){
-        include (dirname(__FILE__)."/../Vendor/vendor/autoload.php");
```

Comment:
We need to include the autoloader.php once. Let's include it in the plugin boot file for the plugin installation.

Reply:
I can change to require_once.
But before refactoring, we included it every time for every helper class use.
Now it's changed to one time, in the moment of creating the ginger client.

Reply:
Ok, let's replace the include with require_once

Reply:
Done